lib/models/quellenreiter_app_state.dart

```diff
@@ -549,28 +552,28 @@ class QuellenreiterAppState extends ChangeNotifier {
     if (db.error != null) {
       return;
     }
-    currentEnemy!.openGame!.statements =
-        await db.getStatements(currentEnemy!.openGame!.statementIds!);
+    currentOpponent!.openGame!.statements =
+        await db.getStatements(currentOpponent!.openGame!.statementIds!);
     notifyListeners();
     return;
   }
 
-  /// Starts an open game with the current enemy.
-  /// only if its the [Player]s turn.
+  /// Starts an open game with the current opponent.
```

Comment:
`/// Starts an open [Game] with [currentOpponent]`

I think this way, it is possible to refactor these variables even in comments. Also If we generate a documentation from these comments, which is possible [see this](https://quellenreiter.github.io/fact-browser-documentation/), these variables will link to their definition etc. 🍬